client/src/app/editview/editview.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { Location } from '@angular/common';
+
+@Component({
+  selector: 'app-editview',
+  templateUrl: './editview.component.html',
+  styleUrls: ['./editview.component.scss']
+})
+export class EditviewComponent implements OnInit {
```

Comment:
In general: Use PascalCase by writing something like this `ng g component edit-view` or `ng g component EditView`